### DISCUSSION TITLE
cannot re-open new keepassxc database

### DISCUSSION THREAD

**Discussion by ChooChooAl** [keepassxreboot/keepassxc#6137] [Q&A] [answered]
running the latest appimage download I was able to import my passwords from LastPass via csv.
All entries seem to work, as I tried a few.
Added the browser add-on for firefox.

Once I closed the app and firefox, I tried to open the app and it keeps telling me there is a missmatch and cannot open the password file.

I did this 3 times.  I know I have the right password, and I'm pointing to the right password file.

Running LInux 20.1 Cinnamon 64bit.
Keepassxc 2.6.4 appimage.

**Comment by droidmonkey** [comment]
Unfortunately there is nothing we can do, you may have mistyped your password originally when creating the database.

**Comment by ChooChooAl** [comment]
![Screenshot from 2021-02-17 17-45-52](https://user-images.githubusercontent.com/26556059/108277946-3a062f80-7148-11eb-8802-533126af1d3b.png)

Sorry for the ambiguity.   I have deleted the password file 3 times, and re-imported my lastpass csv file,  entered the master password, pointed to where the new password database should be and created it.  After checking the data, I exit keypassxc (the application in linux) and tried to reopen the database but each time it says it can't because of a mismatch in authentication.
I know I'm typing the password right, and the location for the db is right.

**Comment by ChooChooAl** [comment, marked as answer]
Figured it out.   At login / opening the database there was a field on the screen called 'key file' (I think that's the name), anyway it was filled in even though I did not set it up.   So I just blanked it, and now it works.

Fixed.

**Reply by droidmonkey** [reply]
That is odd, glad you found the issue!
